docs/content/test/run-tests.md

```diff
@@ -290,6 +303,26 @@ Configure Terraform to use locally-built binaries for `google` and `google-beta`
     TF_LOG=DEBUG TF_LOG_PATH=output.log TF_CLI_CONFIG_FILE="$HOME/tf-dev-override.tfrc" terraform apply
     ```
 
+### Run Tests with VCR Locally
+
+Running tests in "replaying" mode locally can sometimes be useful. In particular, it can allow you to test more quickly, cheaply, and without spinning up real infrastructure, once you've got an initial recording.
+
+It can also be helpful for debugging tests that seem to work locally, but fail in CI in replaying mode.
+   
+Make sure `$VCR_PATH` is set, and points to a local directory which exists, and set `VCR_MODE` appropriately.
```

Comment:
```suggestion
VCR tests record HTTP request/response interactions in cassettes and replay them in future runs without calling the real API.

Running tests in "REPLAYING" mode locally can sometimes be useful. In particular, it can allow you to test more quickly, cheaply, and without spinning up real infrastructure, once you've got an initial recording.

It can also be helpful for debugging tests that seem to work locally, but fail in CI in replaying mode.
   
VCR is controlled via two variables:
- `VCR_MODE`: `REPLAYING` or `RECORDING` mode 
- `VCR_PATH`: Path where recorded cassettes are stored. 

Ensure both variables are configured to properly trigger VCR tests locally.
```

Reply:
I added a bit more details. Let me know your thoughts!